Agents/BuildingFloorAgent/src/main/resources/buildingfloor.odba

```diff
@@ -0,0 +1,19 @@
+[PrefixDeclaration]
+xsd:    http://www.w3.org/2001/XMLSchema#
+bot:    https://w3id.org/bot#
+env: 	https://www.theworldavatar.com/kg/ontobuiltenv/
+twa:    https://www.theworldavatar.com/kg/
+
+[MappingDeclaration] @collection [[
+
+mappingId	building_floor
+target		twa:Building/{strval} env:hasNumberOfFloors twa:NumberOfFloors/{strval} .
+            twa:NumberOfFloors/{strval} a env:FloorCat{storeys_above_ground_cat} ;
```

Comment:
I find defining the rdf type like this is very risky and prone to errors as the values of storeys_above_ground_cat relies heavily on the success of some non-trivial queries/logic executed by the agent.